Lib/test/test_json/test_speedups.py

```diff
@@ -31,6 +31,8 @@ def test(value):
 
 class TestEncode(CTest):
     def test_make_encoder(self):
+        # Issue #6986: The interpreter shouldn't crash in case c_make_encoder()
+        # receives invalid arguments.
```

Comment:
What exactly invalid argument is tested?

Reply:
Is it just insufficient number of arguments? Then why not just call it without arguments? Or call it with 8 valid arguments (it requires 9 arguments)?

Reply:
I guess that before Victor implemented his patch, these specially crafted arguments would cause a crash, and with the patch, nothing bad would happen. (Victor, please correct me if I am wrong.)

Maybe changing the test's name would make it clearer? Or do you have something else in mind?